modules/aleph/lib/AlephClient/AlephClient.class.php

```diff
@@ -0,0 +1,164 @@
+<?php
+/**
+ * @file
+ * Provides a client for the Axiell Aleph library information webservice.
+ */
+
+require __DIR__ . '../../../vendor/autoload.php';
```

Comment:
We need to be 100% on how we autoload classes if we have more than one than one autoload.php in play (we might be good, and might be in trouble)